Gemfile.jruby-1.9.lock.release

```diff
@@ -206,8 +206,9 @@ GEM
       murmurhash3
     logstash-filter-geoip (4.3.1-java)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-grok (3.4.4)
+    logstash-filter-grok (4.0.0)
```

Comment:
Major performance improvement

Reply:
Obligatory, are you sure you want to push a major plugin as the default for a patch release ?

Reply:
LS5 is completely unusable for us without logstash-filter-grok 4.0.0  we had a 90% reduction in grok performance in LS2 vs LS5

Reply:
Yeah, the only reason it's major is that it doesn't support LS 5.4 and below. It'd be a patch release otherwise.

Reply:
There are no new features, it just needs new LS features really.